src/stan/mcmc/chains.hpp

```diff
@@ -117,7 +117,6 @@ class chains {
     using boost::accumulators::tag::tail;
     using boost::accumulators::tag::tail_quantile;
     double M = x.rows();
-    // size_t cache_size = std::min(prob, 1-prob)*M + 2;
```

Comment:
Is there a reason to change anything in chains.hpp?

Reply:
I removed this commented out line after going through the blame - this code has always been there in its commented out form.   why is it here?   maybe to show one alternative - which would use less memory, run faster, and provide approximate as opposed to exact estimates.   I don't know.  so I got rid of this comment, and a few other comments that were clearly outdated.

Reply:
> Is there a reason to change anything in chains.hpp?

the one thing that needed to be changed was to remove the rank-normalized r-hat functions, which weren't being called by anything, since this PR changed the function arguments.